### DISCUSSION TITLE
Docker container works locally but fails in production: Unable to load grader results

### DISCUSSION THREAD

**Discussion by aalrabah** [PrairieLearn/PrairieLearn#10248] [Dev]
We're encountering an issue with our Docker container in production. While it runs without problems locally, we're seeing the following error in the production environment:
"Unable to load grader results"

Has anyone encountered similar issues when moving from local to production? Any suggestions for troubleshooting steps or potential causes we should investigate?

<img width="1186" alt="Screenshot 2024-07-18 at 9 05 30 AM" src="https://github.com/user-attachments/assets/6dd5e59b-f881-4663-b85c-4ef0809b1aaf">

**Comment by jonatanschroeder** [comment]
Have you synced the image in the course syncs page? Can you provide additional information, like the image in use? Did you check the `archive.tar.gz` and `job.tar.gz` files to check if they match what you expected?

**Reply by aalrabah** [reply]
I think the image was not synced in the sync page, upon syncing and waiting for a min, it worked 👍 ... thank you very much @jonatanschroeder
